### ISSUE TITLE
Autogenerated hostnames for Docker containers may be invalid

### ISSUE THREAD

**Issue by nhammond** [StanfordBioinformatics/loom#298]
Docker validates hostnames to be RFC 1123 compliant.

Here we allowed a long LOOM_SERVER_NAME that pushed the host name over the 64-character limit:

```
"msg": "Error creating container: 500 Server Error: Internal Server Error (\"{\"message\":\"invalid hostname format: s0194353-hello-world-workflow-rhl-loom-srvr-mar-06-2017-164540-mysql\"}\")"
```

**Comment by nhammond** [comment]
Can be reproduced locally with
loom server start -e LOOM_SERVER_NAME=a12345678901234567890123456789012345678901234567890123456789012

```
TASK [Start an Elasticsearch container] *************************************************
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "msg": "Error starting container 6c9909b6ea9b683fba5b8daef1ca2479d08ec84fca852f15c55416a81275a477: 500 Server Error: Internal Server Error (\"{\"message\":\"oci runtime error: container_linux.go:262: starting container process caused \\\"process_linux.go:339: container init caused \\\\\\\"invalid argument\\\\\\\"\\\"\\n\"}\")"}
	to retry, use: --limit @/Users/s0198839/.loom-local/playbooks/local_start_server.retry
```

**Comment by nhammond** [comment]
Fixed in 709462d3a1c1325d6e6473dabe9e56f989b8cc75
